### PR TITLE
MEN-4021: Remove passive voice from the Client-installation section.

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: 03.Client-installation/06.Configuration-file/01.Polling-intervals/docs.md [hunk 1]
```diff
@@ -23,12 +23,13 @@ Default value: 28800 seconds (8 hours).
 
 ## How to choose right intervals
 
-The higher the frequency is (i.e. the lower the configuration value), the sooner
-the server will be updated with the client inventory data and the client will
-poll the update install request faster so updates can be deployed more quickly.
+Higher frequency (i.e. the lower the configuration value), the more response the
```
Comment (drewmoseley) [inline 03.Client-installation/06.Configuration-file/01.Polling-intervals/docs.md:26]: The  Mender client is more responsive with higher frequency intervals.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 2]
```diff
@@ -8,10 +8,14 @@ This sections lists all the configuration options in `mender.conf`.
 
 #### ArtifactVerifyKey
 
-Specifies the location of the public key used to verify signed updates, and also
-enables signed-updates-only mode when it is set. If set the client will reject
-incorrectly signed updates, or updates without a signature. See also the section
-about [signing and verification](../../../06.Artifact-creation/07.Sign-and-verify/docs.md).
+When set enables:
```
Comment (drewmoseley) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:11]: When set, the Mender client verifies the following:
* All Artifact installs are signed. If a signature is not provided, then the update is rejected.
* Update signatures are verified using the provided public key.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 3]
```diff
@@ -84,10 +88,10 @@ intervals](../01.Polling-intervals/docs.md).
 
 #### ModuleTimeoutSeconds
 
-An integer that specifies the number of seconds that an update module will be
-allowed to run, before it is considered hanging and killed. The process will
-first be sent a SIGTERM signal, and one minute later, if it has not exited,
-SIGKILL. The default is 4 hours.
+An integer that specifies the number of seconds that an update module is allowed
+to run, before it is considered hanging and killed. The process will first be
```
Comment (drewmoseley) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:92]: considered hung and killed.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 4]
```diff
@@ -84,10 +88,10 @@ intervals](../01.Polling-intervals/docs.md).
...
+sent a SIGTERM signal, and one minute later, if it has not exited, SIGKILL. The
```
Comment (drewmoseley) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:93]: if it has not exited, a SIGKILL signal will be sent.

**Review by drewmoseley — CHANGES_REQUESTED** [review]

**Comment by oleorhagen** [comment]
Thank you @drewmoseley

**Pull request merged**